src/api/index.ts

```diff
@@ -99,3 +101,5 @@ export const tokenListApi: TokenList = createTokenListApi()
 export const erc20Api: Erc20Api = createErc20Api(web3)
 export const depositApi: DepositApi = createDepositApi(erc20Api, web3)
 export const exchangeApi: ExchangeApi = createExchangeApi(erc20Api, web3)
+
+setupAutoconnect(walletApi)
```

Comment:
I don't see this logic here. I think it makes more sense in the App.tsx for example

Reply:
no inside wallet container or something?

Reply:
I really don't want to mess with this anymore. Where it was before caused import race conditions for tests.